metrics/micrometer/src/test/java/com/datastax/oss/driver/internal/metrics/micrometer/MicrometerNodeMetricUpdaterTest.java

```diff
@@ -174,6 +175,10 @@ public void should_create_timer(
     when(profile.isDefined(sla)).thenReturn(true);
     when(profile.getDurationList(sla))
         .thenReturn(Arrays.asList(Duration.ofMillis(100), Duration.ofMillis(500)));
+    when(profile.isDefined(percentiles)).thenReturn(true);
+    when(profile.getDoubleList(percentiles))
+        .thenReturn(
+            Arrays.asList(Double.valueOf(0.75), Double.valueOf(0.95), Double.valueOf(0.99)));
```

Comment:
nit: unneccessary boxing of doubles (and again in the other tests)

Reply:
removed boxing